web/src/assets/main.css

```
@@ -0,0 +1,140 @@
+  /* Reset UL styles to prevent default browser styling */
+  .unapi ul {
+    padding-inline-start: 0;
+    list-style-type: none;
+  }
+
+  /* Reset toggle/switch button backgrounds */
+  .unapi button[role="switch"],
+  .unapi button[role="switch"][data-state="checked"],
+  .unapi button[role="switch"][data-state="unchecked"] {
+    background-color: transparent;
+    background: transparent;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)